setup/react/app/index.js

```diff
@@ -23,7 +23,7 @@ import { MOCK_SERVICE_WORKER } from 'src/const/config';
 import './variables.css';
 import styles from './app.css';
 
-if (MOCK_SERVICE_WORKER) {
+if (!MOCK_SERVICE_WORKER) {
```

Comment:
@eniolam1000752 Since this change was for development purposes, it should be removed